MANIFEST.in

```diff
@@ -1,3 +1,4 @@
 include *.rst *.txt tox.ini LICENSE Makefile
-recursive-include cliquet *.sql
+recursive-include cliquet *.sql *.html
 recursive-include cliquet_docs *.rst *.png *.svg
+prune cliquet_docs/_build/*
```

Comment:
Ah cool, wasn't aware of `prune`.

> exclude all files in the sub-trees starting at directories matching examples/sample?/build—this may exclude files included by the previous two steps, so it’s important that the prune command in the manifest template comes after the recursive-include command